### PR TITLE
Release v0.1.0

### PR DESCRIPTION
**Pull request by jnmt**
This PR is for release v0.1.0.

- Jar file was built with the following commit:
  ToyamaLab/NewSSQL@e060d982b3d5214dfbe2a0399c6314f5cde883d9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +25,18 @@ If you want to use the preview and get the output in `/Users/path/to/outdir`, co
 
 ## Release Notes
 
+### 0.1.0
```
Comment (Kuri-Tai) [inline README.md:28]: IMO: How about writing this release notes in `CHANGELOG.md` to prevent the README.md from becoming huge file in the future?

**Review by Kuri-Tai — COMMENTED** [review]
👍

File: README.md (hunk 1 above)
Comment (jnmt) [reply README.md:28]: Thank you! Good idea. I'll do it right now!

**Comment by jnmt** [comment]
@Kuri-Tai Thank you for the comment! I updated README and CHANGELOG. PTAL.

**Review by Kuri-Tai — APPROVED** [review]
🎉

**Pull request merged**